File: client/my-community/src/api/commit.ts
import service from "@/utils/net"

export const passCommit = (data: string, jwt: string) => {
    return service.request({ url: '/commit/pass', method: 'post', data, headers: { 'Authorization': "Bearer " + jwt } })
}

export const rejectCommit = (data: string, jwt: string) => {
    return service.request({ url: '/commit/reject', method: 'post', data, headers: { 'Authorization': "Bearer " + jwt } })
}

export const communityInfoCommit = (data: string, jwt: string) => {
    return service.request({ url: '/commit/info', method: 'post', data, headers: { 'Authorization': "Bearer " + jwt } })
}

export const userInfoCommit = (data: string, jwt: string) => {
    return service.request({ url: '/commit/info/user', method: 'post', data, headers: { 'Authorization': "Bearer " + jwt } })
}

export const changeCommit = (data: string, jwt: string) => {
    return service.request({ url: '/commit/change', method: 'post', data, headers: { 'Authorization': "Bearer " + jwt } })
}

File: app/lib/api/commit.dart
import 'package:dio/dio.dart';
import 'package:my_community/utils/net.dart';

Future<Response> passCommit(String data, jwt) {
  return Service().getAuthService(jwt).post('/commit/pass', data: data);
}

Future<Response> rejectCommit(String data, jwt) {
  return Service().getAuthService(jwt).post('/commit/reject', data: data);
}

Future<Response> communityInfoCommit(String data, jwt) {
  return Service().getAuthService(jwt).post('/commit/info', data: data);
}

Future<Response> userInfoCommit(String data, jwt) {
  return Service().getAuthService(jwt).post('/commit/info/user', data: data);
}

Future<Response> changeCommit(String data, jwt) {
  return Service().getAuthService(jwt).post('/commit/change', data: data);
}